MeshGeoToolsLib/BoundaryElementsAtPoint.cpp

```diff
@@ -34,17 +35,30 @@ BoundaryElementsAtPoint::BoundaryElementsAtPoint(
     }
     if (node_ids.size() > 1)
     {
-        OGS_FATAL(
+        auto& mesh_nodes =
```

Comment:
Structure like:
```c++
if (node_ids.empty()) OGS_FATAL();
if (node_ids.size() == 1) do_something_for_node(0);
if (!multiple_nodes_allowed) OGS_FATAL();
auto const nearest_node_id = std::min_element(...);
do_something_for_node(nearest_node_id);
```

Reply:
the `multiple_nodes_allowed` must be an explicitly set command line argument.

Maybe it's done in next commit already...